Tests/DataXScenarios/DataXScenarios/Context.cs

```diff
@@ -33,6 +33,7 @@ public class Context
         public const string Seconds = "seconds";
         public const string KernelId = "kernelId";
         public const string InitializeKernelJson = "initializeKernelJson";

```

Comment:
Can you also remove DataBricksToken from Context class and update code where it is used?
public const string DataBricksToken = "dataBricksToken";

Reply:
DataBricks token key is removed now. The references to DataBricksToken from other methods were also removed.